components/Feature/EditAction/index.js

```diff
@@ -10,6 +10,12 @@ const EditAction = ({ action, onActionUpdated, residentName, officerName }) => {
   const [description, setActionDescription] = useState(action.description);
   const [validate, setValidate] = useState(true);
 
+  const containerRef = createRef();
+
+  useEffect(() => {
+    containerRef.current.scrollIntoView({ behavior: 'smooth' });
```

Comment:
Snazzy.

Reply:
**behavior: 'smooth'**

something to live by